update_prevalence.py

```diff
@@ -1239,6 +1244,11 @@ def parse_jhu_place_facts(cache: DataCache, data: AllData, country_by_iso3: Dict
             # has its own entry so turn the combo into just Bristol Bay
             line.Admin2 = "Yakutat"
             line.Population = 604  # from Google
+        if line.Population is None:
+            if line.Admin2 == "Unassigned":
+                line.Population = 0
+            else:
+                raise ValueError(f"Please manually provide population for {line}")
```

Comment:
Example case - mypy forces me to add this `if` statement.  The special case additions were found experimentally by running the script until it completed successfully.